src/components/views/elements/ImageView.tsx

```diff
@@ -290,13 +320,15 @@ export default class ImageView extends React.Component<IProps, IState> {
         let cursor;
         if (this.state.moving) {
             cursor= "grabbing";
-        } else if (this.state.zoom === MIN_ZOOM) {
+        } else if (this.state.maxZoom === this.state.minZoom) {
+            cursor = "pointer";
```

Comment:
Hmm, I think perhaps this should be `default` when clicking the image won't do anything...? `pointer` means it will be a hand e.g. typically used for a link or button, which doesn't seem quite right.